saleor/plugins/sendgrid/plugin.py

```diff
@@ -213,10 +214,17 @@ def notify(self, event: NotifyEventType, payload: dict, previous_value):
         if not self.active:
```

Comment:
Typing for `notify` is incorrect here. It should be a `Union[NotifyEventType, str]`